lib/util/spmconfig.js

```diff
@@ -14,6 +14,21 @@ var unflatten = require('flat').unflatten
 var RC = require('rc-yaml-2')
 var fs = require('fs')
 var os = require('os')
+
+function setReceiverUrlsForRegion (region) {
+  if (region === 'EU' || region === 'eu') {
+    process.env.SPM_RECEIVER_URL = 'https://spm-receiver.eu.sematext.com/receiver/v1'
+    process.env.EVENTS_RECEIVER_URL = 'https://event-receiver.eu.sematext.com'
+    process.env.LOGSENE_RECEIVER_URL = 'https://logsene-receiver.eu.sematext.com'
+    return
+  }
+  if (region === 'US' || region === 'us') {
+    process.env.SPM_RECEIVER_URL = 'https://spm-receiver.sematext.com/receiver/v1'
+    process.env.EVENTS_RECEIVER_URL = 'https://event-receiver.sematext.com'
+    process.env.LOGSENE_RECEIVER_URL = 'https://logsene-receiver.sematext.com'
+    return
+  }
```

Comment:
Can you make US region default, so it works when no region is set?

Reply:
It works when no region is set. US is the default for the config object.